### PR TITLE
Add Content section

### PR DESCRIPTION
**Pull request by liubich**

### CODE REVIEW

**Comment by probil** [comment]
@liubich rebase on latest develop please

File: css/main.css [hunk 1]
```diff
@@ -78,3 +78,59 @@ h1 {
...
+  font-family: "Lato", sans-serif;
+  color: rgb(102, 46, 121);
+  font-weight: bold;
+  letter-spacing: 3px;
+  text-transform: uppercase;
+  text-align: center;
+  margin: 18px 0;
+}
+
+.feature:nth-of-type(2n+1) h6 {
```
Comment (probil) [inline css/main.css]: Looks like a hack. It's a designer's fault that you can't center it properly.
Just remove this class

File: css/main.css [hunk 2]
```diff
@@ -78,3 +78,59 @@ h1 {
...
+
+.feature__icon_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  width: 50px;
+  height: 39px;
+}
+
+.feature__icon_container_little {
```
Comment (probil) [inline css/main.css]: Looks like another hack, just remove it

File: css/main.css [hunk 3]
```diff
@@ -78,3 +78,59 @@ h1 {
...
+  text-align: center;
+  margin: 18px 0;
+}
+
+.feature:nth-of-type(2n+1) h6 {
+  margin-left: 5px;
+}
+
+.feature__icon_container {
+  display: flex;
```
Comment (probil) [inline css/main.css]: flex props are useless here

File: index.html [hunk 4]
```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
...
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Dashboard.png" alt="dashboard" class="feature__icon">
+                        </div>
+                    <h6>Dashboard</h6>
+                    <p class="feature__desc">Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum </p>
+                </div>
+                <div class="feature">
+                    <div class="feature__icon_container">
+                        <img src="./img/Stats.png" alt="stats" class="feature__icon">
```
Comment (probil) [inline index.html]: I can't find `feature__icon` class in css. Do we need it?

File: index.html [hunk 5]
```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
                 </div>
             </div>
         </section>
+        <section class = "sect_content">
+            <div class = "container">
+                <div class="feature">
```
Comment (probil) [inline index.html]: Using `dl/dt/dd` element might be more semantically correct.

https://www.w3schools.com/tags/tag_dd.asp
https://css-tricks.com/utilizing-the-underused-but-semantically-awesome-definition-list/

Read about semantic html.

File: index.html [hunk 6]
```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
                 </div>
             </div>
         </section>
+        <section class = "sect_content">
+            <div class = "container">
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Timer.png" alt="timer" class="feature__icon">
```
Comment (probil) [inline index.html]: 1) For small icons it's better to embed them inline as base64 to make fewer requests to the server
2) Use [TinyPNG](https://tinypng.com/) to compress your icons, 
I've tried it with dashboard items, the result is 2 times smaller `1.4kb -> 711b` but looks the same
3) [Optional] consider `alt="timer icon"`

File: index.html [hunk 7]
```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
...
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Timer.png" alt="timer" class="feature__icon">
+                        </div class="feature__icon_container">
+                    <h6>Designer guff</h6>
+                    <p class="feature__desc">Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum </p>
+                </div>
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Dashboard.png" alt="dashboard" class="feature__icon">
```
Comment (probil) [inline index.html]: [Optional] The `alt` attribute is used for accessibility as well so we should describe in few words what is on the picture. I see a schematic representation of speedometer so `alt="speedometer icon"` looks better for me. I don't know how to explain to a blind person how `dashboard` might look

File: css/main.css [hunk 8]
```diff
@@ -78,3 +78,59 @@ h1 {
 .main_button:hover {
   background: #60d0d0;
 }
+
+.sect_content {
+  padding-top: 70px;
+}
```
Comment (probil) [inline css/main.css]: I think there should be padding bottom as well

File: css/main.css [hunk 9]
```diff
@@ -78,3 +78,59 @@ h1 {
...
+
+.feature:nth-of-type(2n+1) h6 {
+  margin-left: 5px;
+}
+
+.feature__icon_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  width: 50px;
```
Comment (probil) [inline css/main.css]: I suppose `width` is needed just for pixel perfect, right?

File: index.html [hunk 10]
```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
...
+                </div>
+                <div class="feature">
+                    <div class="feature__icon_container">
+                        <img src="./img/Stats.png" alt="stats" class="feature__icon">
+                    </div>
+                    <h6>Detailed stats</h6>
+                    <p class="feature__desc">Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum </p>
+                </div>
+                <div class="feature">
+                        <div class="feature__icon_container_little">
```
Comment (probil) [inline index.html]: re-indent please

File: css/main.css [hunk 11]
```diff
@@ -78,3 +78,59 @@ h1 {
 .main_button:hover {
   background: #60d0d0;
 }
+
+.sect_content {
```
[the remark this answers is not in the document]
Comment (liubich) [reply css/main.css]: @probil, I mean `section` here, section 'Content'. I will change this by `.content-section`

File: css/main.css (hunk 1 above)
Comment (liubich) [reply css/main.css]: Thanks to this fault I found many new features of CSS pseudo-selectors

File: css/main.css (hunk 11 above)
[the remark this answers is not in the document]
Comment (probil) [reply css/main.css]: In any case `.content-section` is too general. Every section is basically content section. You should find a name that describes only this specific section

[the remark this answers is not in the document]
Comment (probil) [reply css/main.css]: > There are only two hard things in Computer Science: cache invalidation and naming things.
> -- Phil Karlton

File: index.html (hunk 5 above)
Comment (liubich) [reply index.html]: @probil, can you add a separate issue for this?

File: index.html (hunk 10 above)
Comment (liubich) [reply index.html]: `Prettier` and `Eslint` have different sights of intendation of ```@font-face``` block in CSS, so I turned Prettier off till I will find a decision of this issue

File: css/main.css (hunk 11 above)
[the remark this answers is not in the document]
Comment (liubich) [reply css/main.css]: > 
> 
> In any case `.content-section` is too general.

Yeah, but it has this name into .psd. First my idea was to name section 'Feature' and I named new branch `add_feat_section`, but later I decided to name section just like its name in .psd file.

File: css/main.css (hunk 9 above)
Comment (liubich) [reply css/main.css]: @probil, yes. the images aren't centered in relation to the text

File: css/main.css [hunk 12]
```diff
@@ -78,3 +78,59 @@ h1 {
 .main_button:hover {
   background: #60d0d0;
 }
+
+.sect_content {
+  padding-top: 70px;
+}
+
+.sect_content .container {
```
[the remark this answers is not in the document]
Comment (liubich) [reply css/main.css]: I think about removing of this `div` container. Will this be semantically correct if I will set a width of content in the class of `section` tag?

[the remark this answers is not in the document]
Comment (liubich) [reply css/main.css]: Can you make a separate issue for overall project BEM methodology styling, please?

File: css/main.css (hunk 11 above)
[the remark this answers is not in the document]
Comment (probil) [reply css/main.css]: psd is not the source of truth. You should give meaningful for your self and for teammates. Designer usually don't spend enough time to find a good name

**Comment by probil** [comment]
@liubich remove icons from the folder please

**Comment by liubich** [comment]
@probil, they are removed in this [commit](https://github.com/liubich/TheRange/pull/16/commits/fc4daa3c7e8a3bef9ded818baf3e10b4da2d69d0)

File: css/main.css (hunk 11 above)
[the remark this answers is not in the document]
Comment (liubich) [reply css/main.css]: Ok. Done.

File: css/main.css (hunk 1 above)
Comment (liubich) [reply css/main.css]: @probil, should I centerise all icons in relation to text?

File: css/main.css (hunk 9 above)
Comment (liubich) [reply css/main.css]: @probil, should I centerise all icons in relation to text?

File: css/main.css (hunk 1 above)
Comment (probil) [reply css/main.css]: @liubich yes

Comment (liubich) [reply css/main.css]: Done

File: css/main.css (hunk 9 above)
Comment (liubich) [reply css/main.css]: @probil, done

**Pull request merged**